com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/FileSystemUtil.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ */
+package com.ca.lsp.cobol.service.utils;
+
+import com.ca.lsp.cobol.service.CopybookServiceImpl;
+import com.ca.lsp.cobol.service.delegates.dependency.CopybookDependencyServiceImpl;
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FilenameUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This is an utility class that provide filesystem related methods consumed by the classes that
+ * rely on the filesystem implementation as {@link CopybookServiceImpl} and {@link
+ * CopybookDependencyServiceImpl}
+ */
+@Slf4j
+@UtilityClass
+public class FileSystemUtil {
+  private final List<String> ALLOWED_EXTENSIONS = Arrays.asList("cpy", "cbl", "cobol", "cob");
+
+  /** @return the representation os based of the FS separator */
+  public static String filesystemSeparator() {
+    return FileSystems.getDefault().getSeparator();
+  }
+
+  /**
+   * @param pathFile NIO path of the file to check
+   * @return true if the path represent a valid file, false otherwise
+   */
+  public static boolean isFileExists(Path pathFile) {
```

Comment:
The method name is syntactically incorrect, it would better to call it something like "checkFileExists"

Reply:
I disagree, following the naming convention "isXXX" or "have/hasXXX" are generally adopted for static true/false checks. I don't see reason to rename it